ghc-mod.cabal

```diff
@@ -193,6 +195,9 @@ Library
                       , transformers-base < 0.5
                       , mtl               < 2.3 && >= 2.0
                       , monad-control     < 1.1 && >= 1
+                      , parsec
+                      , process
+                      , process-streaming
```

Comment:
This doesn't seem to be used for much anything important. I'd rather not depend on it.